website/blog/2018-09-13-decorators.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "Article about Decorators"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2018-09-13 23:00:00
+categories: announcements
+share_text: "Article about Decorators"
+---
+
+> TODO: Better title. Help pls :pray:
+
+Babel 7.1.0 finally supports the new decorators proposal: you can try it out by using the `@babel/plugin-proposal-decorators` plugin 🎉.
+
+<!--truncate-->
+
+## A bit of history
+
+Decorators where [first proposed](https://github.com/wycats/javascript-decorators/blob/696232bbd997618d603d6577848d635872f25c43/README.md) by [Yehuda Kats](https://github.com/wycats) more than three years ago. They have been implemented in TypeScript, which released support for decorators in [version 1.5](https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#typescript-15) (2015) alongside with many ES6 features: modules, destructuring, `let`/`const`, `for ... of` and computed properties.
```

Comment:
where -> were

Think you can ditch the "They have been implemented in" leaving just:

> TypeScript released support for decorators in version 1.5, alongside other ES6 features, like modules, destructuring, `let`/`const`, `for ... of` and computed properties.